.github/workflows/builder_go_slsa3.yml

```diff
@@ -365,6 +371,7 @@ jobs:
         uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # v0.1.15
         if: inputs.upload-tag-name != ''
         with:
+          prerelease: ${{ inputs.prerelease }}
```

Comment:
is this one necessary? If the user provides the tag-name, does it mean they already created the release so we need not do anything?

I'm not familiar enough with `softprops/action-gh-release`, I think.

Reply:
Fixed. 142ad30dc70bfe8dea2628686c55c081ff7b7817

I don't have a strong opinion about this because I usually don't set `upload-tag-name`.